lib/page/spider_base.rb

```diff
@@ -3,12 +3,18 @@
 
 module Page
   class SpiderBase
-    def world
-      World.new.as_json
-    end
-
     def title
       'EveryPolitician: Robots Start Here'
     end
+
+    def countries_sorted
+      world.countries.sort_by(&:slug)
```

Comment:
Sorting by slug is the existing behaviour, but as this is essentially a private page, I don't think it's essential to maintain that order. So if `world.countries` were to return a list ordered by name, I think it would be OK to pass it straight through without resorting here.